### ISSUE TITLE
Error from unspecified start and end climo years in config yaml file

### ISSUE THREAD

**Issue by justin-richling** [NCAR/ADF#149]
### ADF run type

Model vs. Model

### What happened?

Running the ADF without start and end dates crash the package when getting to the ```case_name``` var in the ```perform_analyses``` function in ```lib/adf_diag.py``` script.

### Expected behavior
In the config yaml file it states if the start and end dates are left blank, the package would grab available years from the datasets

### Error
```
Traceback (most recent call last):
  File "./run_adf_diag", line 180, in <module>
    diag.perform_analyses()
  File "/glade/work/richling/ADF/ADF/lib/adf_diag.py", line 788, in perform_analyses
    if syears[case_idx] and eyears[case_idx]:
TypeError: 'NoneType' object is not subscriptable
```

### ADF Hash you are using

5848b92

### What machine were you running the ADF on?

CISL machine

### What python environment were you using?

NPL (CISL machines only)

**Comment by justin-richling** [comment]
I have a working solution and will post a PR when ready

**Comment by justin-richling** [comment]
Closing this, issue was fixed in PR #205
